Description: This makes sure that DESTDIR
  is not overwritten in the Makefile.
Forwarded: doesn't make sense upstream
Author: Ruben Undheim <ruben.undheim@gmail.com>
Index: yosys/Makefile
===================================================================
--- yosys.orig/Makefile	2014-11-08 22:46:32.514901858 +0100
+++ yosys/Makefile	2014-11-08 22:46:32.510901858 +0100
@@ -16,7 +16,7 @@
 # other configuration flags
 ENABLE_GPROF := 0
 
-DESTDIR := /usr/local
+#DESTDIR := /usr/local
 INSTALL_SUDO :=
 
 EXE =
@@ -285,10 +285,10 @@
 	@echo ""
 
 install: $(TARGETS) $(EXTRA_TARGETS)
-	$(INSTALL_SUDO) mkdir -p $(DESTDIR)/bin
-	$(INSTALL_SUDO) install $(TARGETS) $(DESTDIR)/bin/
-	$(INSTALL_SUDO) mkdir -p $(DESTDIR)/share/yosys
-	$(INSTALL_SUDO) cp -r share/. $(DESTDIR)/share/yosys/.
+	$(INSTALL_SUDO) mkdir -p $(DESTDIR)/usr/bin
+	$(INSTALL_SUDO) install $(TARGETS) $(DESTDIR)/usr/bin/
+	$(INSTALL_SUDO) mkdir -p $(DESTDIR)/usr/share/yosys
+	$(INSTALL_SUDO) cp -r share/. $(DESTDIR)/usr/share/yosys/.
 
 manual: $(TARGETS) $(EXTRA_TARGETS)
 	cd manual && bash appnotes.sh
